projects/saturn/src/main/java/io/fairspace/saturn/services/views/ViewRow.java

```diff
@@ -0,0 +1,22 @@
+package io.fairspace.saturn.services.views;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+public class ViewRow {
+    private Map<String, Set<ValueDTO>> data;
+
+    public ViewRow() {
+        data = new HashMap<>();
+    }
+
+    // TODO, make obsolete by ViewStoreReader refactor
+    public Map<String, Set<ValueDTO>> getRawData() {
```

Comment:
I would suggest to return an unmodifiable map, otherwise I am not sure if we need this wrapper